src/starpu/axpy.cc.in

```diff
@@ -139,7 +139,8 @@ void cuda2(void *buffers[], void *cl_args)
 {
 #ifndef STARPU_SIMGRID // Run the code only if this is not a simulation
     // Get arguments
-    auto args = reinterpret_cast<args2_t<T> *>(cl_args);
+    auto args = reinterpret_cast<args2_t *>(cl_args);
+    const T* alpha = reinterpret_cast<const T*>(&args->alpha);
```

Comment:
@Muxas  Is it OK to cast the pointer for scal_t to the proper type for compatibility with the original blas functions?